indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -218,9 +231,24 @@
 }
 
 .abstract-comment-badges {
-    align-self: flex-end;
+    margin-right: auto;
+    align-self: baseline;
     color: $gray;
-    margin-left: 1em;
+}
+
+.i-timeline-item-metadata {
```

Comment:
Add `align-items: baseline` and you can remove `align-self: baseline` from `.abstract-metadata-context` and `.abstract-metadata-action`.